tests/fixtures/addon/defaults-github/tests/dummy/config/ember-cli-update.json

```diff
@@ -0,0 +1,20 @@
+{
+  "schemaVersion": "1.0.0",
+  "packages": [
+    {
+      "name": "ember-cli",
+      "version": "<%= emberCLIVersion %>",
+      "blueprints": [
+        {
+          "name": "addon",
+          "outputRepo": "https://github.com/ember-cli/ember-addon-output",
+          "codemodsSource": "ember-addon-codemods-manifest@1",
+          "isBaseBlueprint": true,
+          "options": [
+            "--ci-provider=github"
```

Comment:
I added a separate fixture for this addon/defaults-github.  Do we expect this `--ci-provider` option to show up in `/tests/dummy/config/ember-cli-update.json`?